modules/dxp/apps/segments/segments-asah-connector-test/src/testIntegration/java/com/liferay/segments/asah/connector/internal/messaging/test/InterestTermsCheckerTest.java

```diff
@@ -51,17 +54,8 @@ public class InterestTermsCheckerTest {
 	public void setUp() throws Exception {
 		_user = TestPropsValues.getUser();
 
-		Bundle bundle = FrameworkUtil.getBundle(InterestTermsCheckerTest.class);
-
-		_interestTermsChecker = _getService(
-			bundle,
-			"(component.name=com.liferay.segments.asah.connector.internal." +
-				"messaging.InterestTermsChecker)");
-
-		Assert.assertNotNull(_interestTermsChecker);
-
 		_asahInterestTermProvider = _getService(
-			bundle,
+			FrameworkUtil.getBundle(InterestTermsCheckerTest.class),
```

Comment:
Hey @regiCesar21, get rid of the `_getService` method and use the `@Inject` instead to retrieve the `AsahInterestTermProvider` reference.